Screen-Sharing-Kotlin/app/src/main/java/com/tokbox/sample/screensharing/ScreenSharingCapturer.kt

```diff
@@ -0,0 +1,76 @@
+package com.tokbox.sample.screensharing
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.os.Handler
+import android.view.View
+import com.opentok.android.BaseVideoCapturer
+
+class ScreenSharingCapturer(private val context: Context, private val contentView: View) :
+    BaseVideoCapturer() {
+    private var capturing = false
+    private val fps = 15
+    private var width = 0
+    private var height = 0
+    private var frame: IntArray? = null
+    private var bmp: Bitmap? = null
+    private var canvas: Canvas? = null
+    private val handler = Handler()
```

Comment:
Which thread does this Handler belong to? I think it belongs to the current thread. Can it run in a different thread?

Reply:
Yes, handler is running in the current thread which is the UI thread. UI-related operations should be executed on the main thread to ensure proper synchronization and avoid UI-related issues